tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/query/GroupByClauseTest.java

```diff
@@ -248,8 +248,6 @@ public void testNegativeCases() {
                 65, 33);
         BAssertUtil.validateError(negativeResult, i++, "incompatible types: expected '[int,int]', found '[int...]'",
                 70, 36);
-        BAssertUtil.validateError(negativeResult, i++, "record literal is not supported for record binding pattern",
```

Comment:
Wasn't this test removed from the bal file?

Reply:
It's removed. `tests/jballerina-unit-test/src/test/resources/test-src/query/group_by_clause_negative.bal` L74:82